nrf_802154_sl/include/fem/simple_gpio/nrf_fem_config.h

```diff
@@ -0,0 +1,127 @@
+#define NRF_FEM_IF_TYPE_SIMPLE_GPIO 1
+
+/**
+ * @brief Configuration parameters for the PA/LNA interface in the "simple gpio" variant
+ */
+typedef struct
+{
+    struct
+    {
+        uint32_t pa_time_gap_us;                /* Time between the activation of the PA pin and the start of the radio transmission. */
+        uint32_t lna_time_gap_us;               /* Time between the activation of the LNA pin and the start of the radio reception. */
+        int8_t   pa_gain_db;                    /* Configurable PA gain. Ignored if the amplifier is not supporting this feature. */
+        int8_t   lna_gain_db;                   /* Configurable LNA gain. Ignored if the amplifier is not supporting this feature. */
+    }                           fem_config;
+
+    nrf_fem_gpiote_pin_config_t pa_pin_config;  /* Power Amplifier pin configuration. */
+    nrf_fem_gpiote_pin_config_t lna_pin_config; /* Low Noise Amplifier pin configuration. */
+
+    uint8_t                     ppi_channels[2];
+
+} nrf_fem_simple_gpio_interface_config_t;
+
+/**
+ * @brief Configures the PA and LNA device interface.
+ *
+ * This function sets device interface parameters for the PA/LNA module.
+ * The module can then be used to control a power amplifier or a low noise amplifier (or both) through the given interface and resources.
+ *
+ * The function also sets the PPI and GPIOTE channels to be configured for the PA/LNA interface.
+ *
+ * @param[in] p_config Pointer to the interface parameters for the PA/LNA device.
+ *
+ * @retval   ::NRFX_SUCCESS                 PA/LNA control successfully configured.
+ * @retval   ::NRFX_ERROR_NOT_SUPPORTED     PA/LNA is not available.
+ *
+ */
+int32_t nrf_fem_simple_gpio_interface_config_set(
+    nrf_fem_simple_gpio_interface_config_t const * const p_config);
+
+/**
+ * @brief Retrieves the configuration of PA and LNA device interface.
+ *
+ * This function gets device interface parameters for the PA/LNA module.
+ * The module can then be used to control a power amplifier or a low noise amplifier (or both) through the given interface and resources.
+ *
+ *
+ * @param[in] p_config Pointer to the interface parameters for the PA/LNA device to be populated.
+ *
+ * @retval   ::NRFX_SUCCESS                 PA/LNA control successfully configured.
+ * @retval   ::NRFX_ERROR_NOT_SUPPORTED     PA/LNA is not available.
+ * @retval   ::NRFX_ERROR_FORBIDDEN         The "simpe gpio" is not currently selected as the FEM type.
+ *
+ */
+int32_t nrf_fem_simple_gpio_interface_config_get(nrf_fem_simple_gpio_interface_config_t * p_config);
+
+/**
+ * @section Timings.
+ */
+
+/** Time in microseconds when PA GPIO is activated before the radio is ready for transmission. */
+#define NRF_FEM_SIMPLE_GPIO_PA_TIME_IN_ADVANCE_US  23
+
+/** Time in microseconds when LNA GPIO is activated before the radio is ready for reception. */
+#define NRF_FEM_SIMPLE_GPIO_LNA_TIME_IN_ADVANCE_US 5
+
+/**
+ * @section Gains.
```

Comment:
This happens in a couple places, ``@section`` is something you don't want to use in a header file,  You probably want ``@name``